movie-browser/src/Card.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+// toogle: https://stackoverflow.com/questions/29913387/show-hide-components-in-reactjs
+class Card extends React.Component{
+    state = {isDetailShown: false}
+
+    handleClick = () => {
+        this.setState(preState => ({isDetailShown: !preState.isDetailShown}));
+    }
+
+    render() {
+        return (
+            <div className="card col-6" onClick={() => this.handleClick()}>
+                <img className="card-img-top" src="https://via.placeholder.com/200x100.png" alt={this.props.title}/>
+                <div className="card-body">
+                    <h5 className="card-title"> { this.props.title } </h5>
+                    <p className="card-text"> { this.props.director } ({ this.props.release_date }) </p>
+                    {this.state.isDetailShown ? <CardDetail { ...(this.props) }/> : null}
+                </div>
+            </div>
+        );
+    }
+}
+
+class CardDetail extends React.Component {
```

Comment:
CardDetail should be in it's own file. You have 2 components in this file. Not ideal

Reply:
In Vue, it was only possible to have one component in one file, but in React, it was possible, so I wasn't sure how far I should to chop it. OK from now one I'll seperate everything.